common/dynamicconfig/collection.go

```diff
@@ -161,6 +161,10 @@ func (c *Collection) GetPingChecks() []pingable.Check {
 	}
 }
 
+func (c *Collection) GetClient() Client {
```

Comment:
is this used anywhere? I'd prefer not exposing the underlying Client if we don't really need to